### PR TITLE
Unify cli commands and flags

### PR DESCRIPTION
**Pull request by xoxys**
- Unify command and flag descriptions
- Unify logs
- Rename `woodpecker-cli [registry|ssecret|...] info` commands to `woodpecker-cli [registry|ssecret|...] show`

### CODE REVIEW

File: cmd/cli/main.go [hunk 1]
```diff
@@ -30,7 +32,23 @@ func main() {
 	})
 
 	app := newApp()
+
+	sortCommandsRecursive(app.Commands)
```
[the remark this answers is not in the document]
Comment (xoxys) [reply cmd/cli/main.go:36]: Sure but then we have to apply it to every single slices or am I wrong?

**Comment by xoxys** [comment]
Ok then Ill adjust it in the cli. Just switched to the most used variant in cli.

File: cmd/cli/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (xoxys) [reply cmd/cli/main.go:36]: Mhm I dont see why we should. Doing it automated in the current way has no performance impact.

[the remark this answers is not in the document]
Comment (xoxys) [reply cmd/cli/main.go:36]: But its totally error prone and I really dont see why we should go through all commands and sort them manually. Ill just drop the sorting from this PR then.

**Review by 6543 — APPROVED** [review]

**Pull request merged**